documentation/modules/ROOT/pages/_partials/compile-and-run.adoc

```diff
@@ -7,7 +7,7 @@ JVM mode::
 [.console-input]
 [source,bash,subs="+macros,+attributes"]
 ----
-./mvnw -DskipTests clean package
```

Comment:
I am not sure if tests will work in all steps that this part is reused. Basically, we are skipping tests for speeding up time. Especially when we do on-site hands-on.

Reply:
I see. While using `./mvnw package` locally, it worked; indeed, it takes less time if we exclude tests, but we might encourage people to work with skipping tests. I am ok with using either version of the command.

Reply:
I just added comments and not asking to change. I am just thinking in terms of speeding up the user experience. Also, since we are covering Continuous Testing, I think we are also encouraging people to write and run tests, but as I said, it was just my point :)